boot/cmdline.go

```diff
@@ -134,50 +134,64 @@ func getBootloaderManagingItsAssets(where string, opts *bootloader.Options) (boo
 	return mbl, nil
 }
 
-// ComposeRecoveryCommandLine composes the kernel command line used when booting
-// a given recovery mode system.
-func ComposeRecoveryCommandLine(model *asserts.Model, system string) (string, error) {
+const (
+	currentEdition = iota
+	candidateEdition
+
+	runBootloader = iota
+	recoveryBootloader
+)
+
+func composeCommandLine(model *asserts.Model, currentOrCandidate, runOrRecovery int, system string) (string, error) {
```

Comment:
The mode name is "recover" so I believe `runOrRecovery` should be `runOrRecover`.

Reply:
hm I guess I should `s/recoveryBootloader/recoverBootloader` too

Reply:
Actually we already have `ModeRun`, `ModeRecover`, so I tweaked the code to pass those identifiers instead.

Reply:
to be clear, RecoveryBootloader is a fine name.  is just the actual mode values that are verbs